Automattic/wp-calypso
# Untangling: Apply new card design in Settings -> Caching

Related to p9Jlb4-eSF-p2#comment-14122

## Proposed Changes

This PR applies the new card design in Settings -> Caching.

As a consequence, the old Performance Optimization card in Server Settings has a slight change. This page is going away soon; I don't think the effort to 100% retain its IA is worth it...

Also, in this PR, to minimize risk, I tried very hard to not touch the lines containing actual logic.

> [!NOTE]  
> Further UX refinements (such as submit button handling) are out of scope for now.

|Settings -> Caching|
|-|
|<img width="340" alt="image" src="https://github.com/user-attachments/assets/edbb3343-ed0f-4feb-9c88-5e9b66abac11">|

|Server Settings (before) |Server Settings (after) |
|-|-|
|<img width="565" alt="image" src="https://github.com/user-attachments/assets/6538d893-28da-47f9-9bbe-7de0ec16f9c9">|<img width="566" alt="image" src="https://github.com/user-attachments/assets/962918dc-dda7-47a6-98ca-1b41ae4b1ecf">|


## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to `/sites`.
2. Click an Atomic site.
3. Go to Settings -> Caching subtab.
4. Verify that the functionality still works.
5. Go to `/sites?flags=-untangling/hosting-menu`.
6. Click an Atomic site.
7. Go to Server Settings tab.
4. Verify that the Performance Optimization card still works.

## Pre-merge Checklist
